### PR TITLE
Add sifrr-dom

### PR DESCRIPTION
**Pull request by aadityataparia**

### CODE REVIEW

File: frameworks/non-keyed/sifrr/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "js-framework-benchmark-stage0",
```
Comment (Freak613) [inline frameworks/non-keyed/sifrr/package.json]: Man, I'm really glad to be your inspiration but... come on.

Comment (aadityataparia) [reply frameworks/non-keyed/sifrr/package.json]: Sorry 🙇

**Comment by aadityataparia** [comment]
This should be ready @krausest ! 🙇

**Comment by krausest** [comment]
Thanks. Results from a first run:
bildschirmfoto 2018-12-20 um 20 48 09

**Pull request merged**